front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/FollowedItems/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Multiloc } from 'typings';
+
+import useProjectsMini from 'api/projects_mini/useProjectsMini';
+
+import useLocalize from 'hooks/useLocalize';
+
+import EmptyState from '../_shared/EmptyState';
+import ProjectCarrousel from '../_shared/ProjectCarrousel';
+
+import messages from './messages';
+import Settings from './Settings';
+
+interface Props {
+  titleMultiloc: Multiloc;
+}
+
+const FollowedItems = ({ titleMultiloc }: Props) => {
+  const localize = useLocalize();
+  const { data, hasNextPage, fetchNextPage } = useProjectsMini({
+    endpoint: 'for_followed_item',
+  });
+  const projects = data?.pages.map((page) => page.data).flat();
+
+  if (!projects) return null;
```

Comment:
Should we return a spinner while the projects are loading?

Reply:
I decided not to because it's possible that the number of projects is zero, which would lead us to hide the widget. In this situation you would have a spinner that then disappears. But if you strongly feel that this is preferable to having no spinner at all, I can still add it.

Reply:
Hmm no I see your point! Fine to leave it out then :+1: Thanks for explaining :)